lib/presentation/extensions/send_file_extension.dart

```diff
@@ -418,6 +427,35 @@ extension SendFileExtension on Room {
     return eventId;
   }
 
+  Future<ImageFileInfo> convertHeicToJpgImage(ImageFileInfo fileInfo) async {
+    final convertedFilePath =
+        StorageDirectoryManager.instance.convertFileExtension(
+      fileInfo.filePath,
+      'jpg',
+    );
+    final newPath = await HeifConverter.convert(
+      fileInfo.filePath,
+      output: convertedFilePath,
+    );
+    Logs().d('sendFileEvent::Heic converted to jpg', newPath);
+    if (newPath != null) {
+      final newConvertedFile = File(convertedFilePath);
+      fileInfo = ImageFileInfo(
+        newConvertedFile.path.split("/").last,
+        newConvertedFile.path,
+        await newConvertedFile.length(),
```

Comment:
What happens if this func fails?

Reply:
if the function fails it just continue the execution

Reply:
try/catch already